src/common/core/core.js

```diff
@@ -3911,7 +3942,406 @@ define([
                 core.loadOwnMembers(node, setName, callback);
             }
         };
+
+        /**
+         * Renames the given pointer of the node.
```

Comment:
`Renames the given pointer stored at the given node. Only has an effect if "oldName" is part of node.getOwnPointerName(node).` Same for the ones below.